build/common.project.props

```diff
@@ -18,6 +18,7 @@
     <NETCoreTargetFrameworks Condition=" '$(RequiresSigningXplatAPIs)' != 'true' ">netcoreapp2.1</NETCoreTargetFrameworks>
     <NETCoreTargetFrameworks Condition=" '$(RequiresSigningXplatAPIs)' == 'true' ">netcoreapp2.1;netcoreapp5.0</NETCoreTargetFrameworks>
     <NETCoreTestTargetFrameworks>netcoreapp3.1;netcoreapp5.0</NETCoreTestTargetFrameworks>
```

Comment:
I assume `netcoreapp3.1` is being kept here because some of our assemblies target both `netstandard2.0` and `net5.0`, and `netcoreapp3.1` is being used to test the `netstandard2.0` binaries?

Reply:
yup.

Reply:
Seems `NETCoreTestTargetFrameworks `is not used anywhere?

Reply:
Yeah, great point @heng-liu , I missed that NETCoreTestTargetFrameworks. 

GH hadn't refreshed the page and I just merged :(. 

I have another build related clean-up, I'll handle it there.